API_Guidelines.md

```diff
@@ -102,6 +103,37 @@ Payload
 }
 ```
 
+### GET single record using a query
+#### Request
+URL
+:   `apiHost.com/movies?title=Goodfellas&year=1980`
+
+Ember Data method
+:   `queryRecord('movie', {
+		title: 'Goodfellas',
+		year: '1980' })`
+
+### Why?
+
+> To get a single record based on parameter criteria.
```

Comment:
how do you ensure that your query only matches 1 record? maybe @jduuuubs can weigh in